test.xunit.runner.visualstudio.testadapter/RunSettingsHelperTests.cs

```diff
@@ -0,0 +1,177 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Runner.VisualStudio.TestAdapter;
+
+namespace test.xunit.runner.visualstudio.testadapter
```

Comment:
It auto-added and the default namespace in csproj is lowercase. The other file uses Camel Case. I will change it just in case.